proposals/open-telemetry/README.adoc

```diff
@@ -1,198 +1,47 @@
 
 = MicroProfile OpenTelemetry
 
-[IMPORTANT]
-====
-The following content is moved from the original https://docs.google.com/document/d/1-mFPmFjxmOx29rDrr1qBtpOXmGZtqVquF0_BvpklO5s/edit#heading=h.q5u44wt1zwb9[Google Doc document]:
-====
+This specification is to adopt https://opentelemetry.io/ [OpenTelemetry] from CNCF to allow MicroProfile Applications benefit from 
+OpenTelemetry and enable Jakarta RESTful Web Services automatically being traced.
+
 == OpenTracing move to OpenTelemetry
+With the merge of OpenTracing and Open Census, OpenTelemetry is the new future. 
+OpenTelemetry consists of Tracing, Logging and Metrics support. This reposition is to start with Tracing part first with the potential to add logging support.
+
+== Multiple modules
+Since this specification potentially expands to support logging from OpenTelemetry, it will use multi module structure. 
+For the time being, the main focus is OpenTelemetry Tracing. When OpenTelemtry Logging is ready for production, we will then evaluate the need to pull in the OpenTelemetry Logging.
+
+== Directly adopt the APIs from OpenTelemetry Tracing
+
+This specification directly expose the Tracing APIs from OpenTelemetry and then provide CDI and Jakarta RESTful Web Services integration.
+The https://github.com/open-telemetry/opentelemetry-java/tree/main/api/all/src/main/java/io/opentelemetry/api/trace Tracer APIs from OpenTelemetry will be exposed.
+This specification does not plan to reinvent the wheels but rather exposes OpenTelemetry APIs due to the following reasons:
+* OpenTelemetry is a specification under CNCF 
+* OpenTelemetry is widely adopted by different vendors and languages
+* OpenTelemetry has Java API and annotations that are readily to be used
+* OpenTelemetry is active and open for the collaboration with us
+* OpenTelemetry is platform technology and we use the effort to make Java libraries to be used directly instead of reinventing our own APIs
```

Comment:
Add a new reason
```suggestion
* OpenTelemetry is platform technology and we use the effort to make Java libraries to be used directly instead of reinventing our own APIs
* OpenTelemetry has broad vendor support. Many APMs are contributing to the spec, providing a guarantee that traces can be used out of the box, with minimal integrations. 

```

Reply:
Do you not want @brunobat point to be added, @Emily-Jiang ?